api/main.py

```diff
@@ -94,6 +97,32 @@ def human(id):
     # <a href=\"https://stampy.ai/?state=6207&question=What%20is%20%22superintelligence%22%3F\">
     text = re.sub(r'<a href=\\"/\?state=(\d+.*)\\">', r'<a href=\"https://aisafety.info/?state=\1\\">', r.text)
 
+    LANGCHAIN_API_KEY = os.environ["LANGCHAIN_API_KEY"]
```

Comment:
can you move these to `env.py` and import them from there?

Reply:
modified it in new [commit](https://github.com/StampyAI/stampy-chat/pull/132/commits/d94abb7d9908728577beb7377a1749368d2860e1)